serialization-avro/src/main/avro/giftcard.command.RedeemCard.avsc

```diff
@@ -0,0 +1,19 @@
+{

```

Comment:
Another question: does the file name become the package name upon generation? So, would there be a generated `RedeemCard` in a `command` folder, which in turn is in a `giftcard` folder?

Reply:
Nope. The generator uses Avro attributes namespace and name as coordinates and translates it to package/classname. The file name doesn't matter at all, I used it for better visibility for the humans.